docs/intents-intro.md

```diff
@@ -19,24 +19,40 @@ FDC3 Intents define a standard set of verbs that can be used to put together com
 ## Using Intents
 Combined with [Context Data](context-intro.md) and [App Directory](appd-intro.md) standards, Intents enable rich service discovery on the desktop. For example:
 
-### Directing a market data platform to show a chart
+### Ask for a chart to be displayed
 ```javascript
-fdc3.open("my-platform","ViewChart",{
- type:"fdc3.instrument",
+const result = await fdc3.raiseIntent("ViewChart", {
+ type: "fdc3.instrument",
  name: "IBM",
- id:{
+ id: {
     ticker:"ibm"
-    }
-  });
+  }
+});
 ```
 
-### Discovering apps that can start a chat
+### Ask a specific application to display a chart
 ```javascript
-fdc3.raiseIntent("StartChat",{
- type:"fdc3.contact",
+const result = await fdc3.raiseIntent("ViewChart", {
+ type: "fdc3.instrument",
+ name: "IBM",
+ id: {
+    ticker:"ibm"
+  }
+}, "market-data-app");
+```
+
+### Find applications that can start a chat
+```javascript
+const intentApps = await fdc3.findIntent("StartChat");
```

Comment:
The `findIntents` call returns an object, not the actual apps :
```
// I know 'StartChat' exists as a concept, and want to know more about it ...
const appIntent = await agent.findIntent("StartChat");
// returns a single AppIntent:
// {
//     intent: { name: "StartChat", displayName: "Chat" },
//     apps: [{ name: "Skype" }, { name: "Symphony" }, { name: "Slack" }]
// }

// raise the intent against a particular app
await agent.raiseIntent(appIntent.intent.name, context, appIntent.apps[0].name);
```

Reply:
I know, but this isn't the API docs, I wanted to be concise and unspecific and get the idea across of what it turns. Do you think I should change it? It could still be an object with that name in my example.

Reply:
Cool - just checking.